src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/CvlReportController.kt

```diff
@@ -0,0 +1,105 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.privateApi
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.response.LastMinuteHandoverCaseResponse
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.response.UpcomingReleasesWithMonitoringConditionsResponse
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.Tags
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.reports.LastMinuteHandoverCaseService
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.reports.UpcomingReleasesWithMonitoringConditionsReportService
+
+@RestController
+@Tag(name = Tags.REPORTS)
+@RequestMapping(value = ["/cvl-report"], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
had to call this `cvl-reports` as anything on `reports` automatically gets routed by the dpr library 😠